server/src/index.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express');
 const path = require('path');
+const questions = require('../routes/questions.js');
```

Comment:
Там index.js был не просто так, это для удобства ре-экспортов. Поэтому здесь лучше сделать 
```suggestion
const routes= require('../routes/index.js');
```
Чтобы потом использовать как `routes.questions`, все роуты будут собраны в один объект.